pkg/server/admin.go

```diff
@@ -1040,14 +1041,44 @@ func (s *adminServer) Cluster(
 func (s *adminServer) Health(
 	ctx context.Context, req *serverpb.HealthRequest,
 ) (*serverpb.HealthResponse, error) {
+	isLive, err := s.server.nodeLiveness.IsLive(s.server.NodeID())
```

Comment:
For what it's worth, comments more clearly explaining the reason for using `IsLive` vs `IsHealthy` here or in node_liveness.go would be nice. To really be sure I had to look at the commit message of bcc6d0b085928fb3cb5faaf2edd074edfa558b2b

Reply:
Done (added to `node_liveness.go`) let me know if this is what you had in mind.